docs/pages/what-is-mud.mdx

```diff
@@ -85,7 +84,7 @@ Store can either be used on the same contract running your business logic (trade
 
 [diagram: store w/ logic, multiple logic contracts talking to Store]
 
-Store uses a custom storage encoding called ********************Tight-coder.******************** It makes Store cheaper to use than regular Solidity for data with more than one dynamic field (eg: two arrays in a struct), and roughly similarly expensive for other data structures.
+Store uses a custom storage encoding called **\*\*\*\***\*\*\*\***\*\*\*\***Tight-coder.**\*\*\*\***\*\*\*\***\*\*\*\*** It makes Store cheaper to use than regular Solidity for data with more than one dynamic field (eg: two arrays in a struct), and roughly similarly expensive for other data structures.
```

Comment:
why do we have so many `******` around this haha

Reply:
might have to do with exporting .md from notion